gnome-extra/nautilus-dropbox/nautilus-dropbox-2019.02.14.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+GNOME2_LA_PUNT="yes"
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit autotools python-single-r1 gnome2
+
+DESCRIPTION="Dropbox Nautilus Extension"
+HOMEPAGE="https://github.com/dropbox/nautilus-dropbox"
+SRC_URI="https://linux.dropboxstatic.com/packages/${P}.tar.bz2"
+
+LICENSE="GPL-3 CC-BY-ND-3.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug"
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RDEPEND="${PYTHON_DEPS}
+	gnome-base/nautilus
+	net-misc/dropbox
+	>=dev-libs/glib-2.14
+	$(python_gen_cond_dep 'dev-python/pygobject:3[${PYTHON_MULTI_USEDEP}] ')
+	"
+
+DEPEND="${RDEPEND}
+	virtual/pkgconfig
+	$(python_gen_cond_dep 'dev-python/docutils[${PYTHON_MULTI_USEDEP}] ')"
+
+src_prepare() {
+	gnome2_src_prepare
+
+	# use system dropbox
+	sed \
+		-e "s|~/[.]dropbox-dist|${EPREFIX}/opt/dropbox|" \
+		-e "s|\(DROPBOXD_PATH = \).*|\1\"${EPREFIX}/opt/dropbox/dropboxd\"|" \
+		-i dropbox.in || die
+	# use system rst2man
+	epatch "${FILESDIR}"/${PN}-2019-system-rst2man.patch
```

Comment:
`epatch` is banned from being used in ebuilds, use `eapply` or global `PATCHES=()` array instead.